src/encoded/static/components/cart/status.js

```diff
@@ -137,7 +137,7 @@ class CartStatusComponent extends React.Component {
                     <DropdownMenu label="cart-control">
                         {menuItems}
                     </DropdownMenu>
-                    {this.state.shareModalOpen ? <CartShare userCart={savedCartObj} closeShareCart={this.closeShareCart} /> : null}
+                    {this.state.shareModalOpen ? <CartShare disableFooterSubmitBtn={inProgress} userCart={savedCartObj} closeShareCart={this.closeShareCart} /> : null}
```

Comment:
`<CartShare>` doesn’t use `disableFooterSubmitBtn`, and it doesn’t need to because you’re already retrieving the store `inProgress` state.